engine/telegraph.lua

```diff
@@ -309,23 +311,32 @@ function Telegraph:telegraphRenderXPosition(index)
   return result
 end
 
-function Telegraph:attackStartFrame()
+function Telegraph:attackAnimationStartFrame()
+  -- Traditionally this would be right after #card_animation
+  -- but PA doesn't keep attacks in the telegraph quite as long.
+  -- This is so they can go over the network sooner for rollback.
+  -- As a result we need to start the attack animation sooner so it has time
+  -- to get to and stay in the telegraph
```

Comment:
At first this confused me a bit as I didn't touch the original games in a long time and the comment didn't make a lot of sense as it has no direct relation to network. I had to watch a YT video and look at the attack animation in TA again to make sense of it.
I tried to rephrase it in a way that I believe contextualizes slightly better.
```suggestion
function Telegraph:attackAnimationStartFrame()
  -- In Tetris Attack the attack animation only happens after the card_animation and popFX
  -- In PA garbage is removed from telegraph early in order to afford the 1s desync tolerance for online play
  -- to compensate, both attacks and telegraph are shown earlier so they are shown long enough and early enough
  -- an online opponent will typically be 15-30 frames behind the local player
  -- their attacks being rendered immediately produces a decent compromise on visuals
```